icons/id-card.svg

```diff
@@ -0,0 +1,17 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="8" cy="10" r="1" />
+  <rect x="2" y="5" width="20" height="14" rx="2" />
+  <path d="M14 14h4" />
+  <path d="M6 15a2.5 2.5 0 0 1 4 0" />
```

Comment:
```suggestion
  <path d="M6 15h4" />
```

Reply:
Not a fan, that's exactly the issue we were trying to avoid ("head looks detached from the body").

Reply:
I really dislike it as well, I just generally don't like our human icons, maybe we can get some sort of animal related outline there instead?

Reply:
Perfect! ![image](https://github.com/lucide-icons/lucide/assets/17746067/be98e19b-06fc-43d0-bac1-53f6f46d9996)

Reply:
Best I could come up with:

<img width="925" alt="image" src="https://github.com/lucide-icons/lucide/assets/25524993/729edc44-2dcf-4815-8c6e-bb9589bb3d62">

Reply:
How about just adding a silhouette like this?
![image](https://github.com/lucide-icons/lucide/assets/17746067/f2760750-6f24-4416-8093-c5964b529530)

Reply:
Like a playing figure, that's not a bad idea.